vortex-array/src/arrays/primitive/compute/is_constant.rs

```diff
@@ -32,7 +32,7 @@ impl IsConstantFn<&PrimitiveArray> for PrimitiveEncoding {
 
 // Assumes any floating point has been cast into its bit representation for which != and !is_eq are the same
 // Assumes there's at least 1 value in the slice, which is an invariant of the entry level function.
-fn compute_is_constant<T: NativePType, const WIDTH: usize>(values: &[T]) -> bool {
+pub fn compute_is_constant<T: NativePType, const WIDTH: usize>(values: &[T]) -> bool {
```

Comment:
I would love to have generic const exprs to be able to derive width inside the function. Right now it's the callers responsibiltiy